src/LightSm.csx

```diff
@@ -31,50 +31,33 @@ void PrintSmInfo(StateMachine sm)
     BehaviorDescriber describer = new(singleLineFormat: true);
 
     InitialState rootInitialState = sm.ChildType<InitialState>();
-    Console.WriteLine($"Root initial state behaviors:");
-    PrintBehaviors(rootInitialState, describer);
-    Console.WriteLine($"");
-
-    // The below code will visit all states in the state machine.
-    // A more complete solution would also visit pseudo states like initial, choice, enter and exit points.
-    // Let's start simple for now and ignore pseudo states.
-    sm.VisitTypeRecursively((State state) =>
-    {
-        Console.WriteLine($"State {state.Name} behaviors:");
-        // could use name to generate a test function name
-
-        PrintBehaviors(state, describer);
-        Console.WriteLine("");
-    });
-}
-
-// Note that this function takes a vertex, which is a base class for states and pseudo states.
-static void PrintBehaviors(Vertex vertex, BehaviorDescriber describer)
-{
-    foreach (var behavior in vertex.Behaviors)
-    {
-        // To learn how to inspect a behavior for transitions, guards, ...
-        // see https://github.com/StateSmith/StateSmith/blob/41d7b6bb663dac3d99c0223a15dda3638cd21548/src/StateSmith/SmGraph/BehaviorDescriber.cs#L8
-        Console.WriteLine("    " + describer.Describe(behavior));
-    }
-}
-
-
-
-
 
+    // Imports
+    Console.WriteLine($"import {{jest}} from '@jest/globals';");
+    Console.WriteLine($"import {{ {sm.Name} }} from './{sm.Name}.js';");
+    Console.WriteLine($"");
 
+    // beforeEach
+    Console.WriteLine($"beforeEach(() => {{");
+    Console.WriteLine($"    jest.clearAllMocks();" );
+    Console.WriteLine($"}});");
+    Console.WriteLine($"");
 
+    // TODO is it a valid assumption to assume that every state machine has at least two vertices?
+    // TODO is it a valid assumption to assume that the zeroth vertex is the entry vertex, and the 
+    //      next vertex is the next reachable state from the vertex?
+    NamedVertex firstState = sm.GetNamedVerticesCopy()[1];
```

Comment:
Every useful state machine will at least have a root initial state. We can get access to that with the `rootInitialState` variable above.

Then it gets more interesting. The initial state is a pseudo state and it can go to other pseudo states (like a choice state) before it lands in a regular NamedVertex.

For now, we can restrict ourselves to diagrams where there is only a single transition from the initial state to a NamedVertex with this code: `NamedVertex firstState = (NamedVertex)rootInitialState.TransitionBehaviors().Single().TransitionTarget;`

The `.Single()` call is C# linq. It will throw if there is not exactly 1 transition behavior. Not perfect, but not bad for now.

Here's a better explanation of all the things that are possible with pseudo states: https://github.com/StateSmith/StateSmith/wiki/Choice-Pseudo-States

It becomes an interesting tree/graph problem.